frontend/src/app/messages/messages.component.html

```diff
@@ -0,0 +1,8 @@
+<p *ngIf="showMessages">
+  <ngb-alert
+    *ngFor="let message of messages$ | async"
+    [type]="message.type ==='error' ? 'warning': message.type"
```

Comment:
In my opinion, you're missing two spaces in this line. I'm almost sure that `ng lint` would not allow this if it was in .ts. file.